ui/src/components/NodePartitionTable.js

```diff
@@ -118,12 +118,17 @@ const NodePartitionTable = ({ instanceIP }: { instanceIP: string }) => {
               nodeFSSizeResult.data.result,
               alerts,
             );
+          } else if (
+            nodeFSUsageResult.error ||
+            nodeFSSizeResult.error ||
+            alerts.error
+          ) {
+            throw new Error();
```

Comment:
This should be part of those functions : `queryNodeFSUsage`, `queryNodeFSSize`, `getAlerts`, if getAlerts is used by another component you may want to implement a getNodePartitionAlert that handle this